### ISSUE TITLE
include more pacakges

### ISSUE THREAD

**Issue by PeerHerholz** [PeerHerholz/BIDSonym#9]
### Summary

There are some other cool defacing tools out there we should include in BIDSonym.
For example: [DeepDeface](https://github.com/AKhazane/DeepDeface). 

Further software will be collected and documented in this issue, with the participating (and all other) folks deciding which tools will be included.

**Comment by PeerHerholz** [comment]
Just discovered that [FSL](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSL) now also has a defacing tool which is based on the one from [UK Biobank](https://www.sciencedirect.com/science/article/pii/S1053811917308613). It's already included being called fsl_deface, however apparently not working properly (e.g., [see here](https://www.jiscmail.ac.uk/cgi-bin/webadmin?A2=FSL;4fea3c4.1903)). Once it has been tested and working properly, we should include it, as we already have FSL in the docker container.

**Comment by PeerHerholz** [comment]
We should also add [mridefacer](https://github.com/mih/mridefacer)!

**Comment by PeerHerholz** [comment]
[mridefacer](https://github.com/mih/mridefacer) was added [here](https://github.com/PeerHerholz/BIDSonym/commit/ec61d3d602980a889510041540e85d340d4d6428).

**Comment by yarikoptic** [comment]
yes --- would be great to see `fsl_deface` be interfaced by BIDSonym!
